tools/kete.py

```diff
@@ -30,13 +30,15 @@
 
 try:
     from tuhi.svg import JsonSvg
+    from tuhi.png import JsonPng
```

Comment:
honestly, I wouldn't mind just shoving the png bits into `tuhi.svg`. Or possibly renaming it to `tuhi.image` or something.